ansible_collections/arista/avd/molecule/eos_cli_config_gen_v4.0/inventory/host_vars/router-bgp-evpn.yml

```diff
@@ -186,3 +186,20 @@ router_bgp:
           peer_group: MLAG-IPv4-UNDERLAY-PEER
       redistribute_routes:
         - source_protocol: connected
+    - name: TENANT_A_PROJECT03
+      router_id: 192.168.255.3
+      rd: "192.168.255.3:13"
+      evpn_multicast: true
+      evpn_multicast_address_family:
+        ipv4:
+          disabled: true
+          transit: true
```

Comment:
probably worth adding tests where these are set to false to see the output as well

Reply:
Have now done so.  Also added documentation as suggested.